plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -130,7 +130,7 @@ class Yarn2(override val descriptor: PluginDescriptor = Yarn2Factory.descriptor,
 
         val packageInfos = getPackageInfos(workingDir)
         val packageHeaders = parsePackageHeaders(packageInfos)
-        val packageDetails = queryPackageDetails(
+        val packageDetails = getRemotePackageDetails(
```

Comment:
I'm not against the renaming, but the consistency argument is a bit shady, as the signature of the function is different for the other Node package managers.

Maybe just say "Make the name consistent with *the similar funtions* in other Node package managers."